doc/getting-started.md

```diff
@@ -0,0 +1,49 @@
+# Getting Started
+
+ct is a tool that will consume a Container Linux Config and produce a JSON file
+that can be given to a Container Linux machine when it first boots to set the
+machine up. Via this config a machine can be told to create users, format the
+root filesystem, set up the network, install systemd units, and more.
+
+Container Linux Configs are YAML files conforming to ct's schema. For more
+information on the schema, take a look at [doc/configuration.md][1].
+
+As a simple example, let's use ct to set the authorized ssh key for the core
+user on a Container Linux machine.
+
+```yaml
+passwd:
+  users:
+    - name: core
+      ssh_authorized_keys:
+        - ssh-rsa AAAAB3NzaC1yc...
+```
+
+In this above file, you'll want to set the `ssh-rsa AAAAB3NzaC1yc...` line to be
+your ssh public key (which is probably the contents of `~/.ssh/id_rsa.pub`, if
+you're on Linux).
+
+If we take this file and give it to ct:
+
+```
+derek@proton ~> ./bin/ct --in-file example.yaml --out-file ct-results
+derek@proton ~> cat ct-results 
+{"ignition":{"version":"2.0.0","config":{}},"storage":{},"systemd":{},"networkd":{},"passwd":{"users":[{"name":"core","sshAuthorizedKeys":["ssh-rsa AAAAB3NzaC1yc..."]}]}}
```

Comment:
You can use `--pretty` to format the JSON nicely. Not sure if we will want to actually do that here.